configurator/configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/ResultImpl.java

```diff
@@ -71,12 +75,12 @@ public Status getStatus() {
     }
 
     @Override
-    public Optional<Throwable> getBadOutcome() {
-        return Optional.ofNullable(badOutcome);
+    public Optional<ConfiguratorException> getError() {
+        return Optional.ofNullable(error);
```

Comment:
Is there going to be an easy way for anyone using this to identify what went wrong in the process? For example, if the user does not have permissions to perform the action, is there an easy way to check the error and so whoever calls it can return back unauthorized?